Add multi-instance RPMsg and IPC Service (#36120)
zephyrproject-rtos/zephyr: samples/subsys/ipc/ipc_service/README.rst

.. _IPC_Service_sample:

> IPC Service sample Application
> ##############################

Reviewer: ```suggestion
IPC Service
###########
```
when adding a sample or library, do not mention sample or library in the title of the page (https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/doc_styleguide.html#titles)

Author: Thanks. Fixed.

> ################################

Reviewer: Title underline too long.

Author: Fixed.

Overview
********

IPC Service is an abstraction layer. It needs the corresponding backend registered for proper operation.

> Any IPC machanism can by used as a backend. In this sample it is a multiple instance of RPMsg.

Reviewer: ```suggestion
You can use any IPC mechanism as a backend.
In this sample, it is a multiple instance of RPMsg.
```

This application demonstrates how to use IPC Service in Zephyr. It is designed
to demonstrate how to integrate IPC Service with Zephyr both from a build

> perspective and code.

Reviewer: General comment: Start each sentence on a new line.

Author: Fixed.

> This application demonstrates how to use IPC Service in Zephyr. It is designed
> to demonstrate how to integrate IPC Service with Zephyr both from a build
> perspective and code.

Reviewer: ```suggestion
This sample demonstrates how to integrate IPC Service with Zephyr.
```

Building the application for nrf5340dk_nrf5340_cpuapp
*****************************************************

.. zephyr-app-commands::
   :zephyr-app: samples/subsys/ipc/ipc_service
   :board: nrf5340dk_nrf5340_cpuapp
   :goals: debug

> Open a serial terminal (minicom, putty, etc.) and connect the board with the

Reviewer: ```suggestion
Open a serial terminal (for example Minicom or PuTTY) and connect the board with the
```
Do not use latin-based abbreviations, such as etc.

following settings:

- Speed: 115200
- Data: 8 bits
- Parity: None
- Stop bits: 1

> Reset the board and the following message will appear on the corresponding
> serial port, one is master another is remote:

Reviewer: ```suggestion
When you reset the development kit, the following messages (one for master and one for remote) will appear on the corresponding serial ports:
```
